src/_sass/_site.scss

```diff
@@ -408,7 +421,7 @@ main .content {
   }
 
   a {
-    color: $site-color-card-link;
+    color: $flutter-color-dark-blue;
```

Comment:
Did you mean to change this color? This is darker than before, I would prefer to keep the style we currently have.

Reply:
I did. That color was not one included in the [Dart brand guidelines](go/dart-brand-guidelines), so I wanted to make it consistent. It is as dark as the header.